src/api/airtableApi.jsx

```diff
@@ -94,3 +94,28 @@ export const create = async newRecord => {
     console.log(error);
   }
 };
+
+export const updateRecords = async titles => {
+  try {
+    const response = await fetch(tableUrl, {
+      body: JSON.stringify({
+        records: titles.map(title => {
+          const { id: _, ...titleWithoutId } = title;
```

Comment:
This omits the `id` property in the blog title so that the rest of the title can be passed into the `fields` of the request.